test/components/DateInput_spec.jsx

```diff
@@ -58,6 +58,46 @@ describe('DateInput', () => {
       });
     });
 
+    describe('screen reader message', () => {
```

Comment:
Just had another look at the tests. I'm unsure what other components I could test?

All the tests files use enzymes shallow rendering, so I can only test that level of the component. The changes I've added to them are pretty simple. All they do is take a screenReaderMessage prop and pass it to a child component. This prop is passed all the way down until it gets to the `DateInput` component, which the above tests cover.

Happy to add more tests but need some further instruction, thanks.